legacy/jacorb/src/main/java/org/jboss/as/jacorb/JacORBSubsystemDefinitions.java

```diff
@@ -492,11 +489,16 @@
             .setValidator(new EnumValidator<CallerPropagationValues>(CallerPropagationValues.class, true, true))
             .setAllowExpression(true).build();
 
+    // This is added to enable running many Wildfly instances with JacORB legacy subsystem enabled on one host. To enable
+    // this it is necessary to modify legacy configuration.
+    static final SimpleAttributeDefinition PERSISTENT_SERVER_ID = new SimpleAttributeDefinitionBuilder(
+            Constants.ORB_PERSISTENT_SERVER_ID, ModelType.STRING, true).setDefaultValue(new ModelNode().set("1"))
+            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES).build();
```

Comment:
This needs a transformer to reject it if defined when sending to a legacy slave.